code/pagetypes/ForumHolder.php

```diff
@@ -208,26 +208,21 @@ public function Breadcrumbs($maxDepth = 20, $unlinked = false, $stopAtPageType =
 				case 'search':
 					return '<a href="' . $this->Link() . '">' . $this->Title . '</a> &raquo; ' . _t('SEARCHBREADCRUMB', 'Search');
 				case 'memberlist':
-					return '<a href="' . $this->Link() . '">' . $this->Title . '</a> &raquo; ' . _t('MEMBERLIST', 'Member List');				
+					return '<a href="' . $this->Link() . '">' . $this->Title . '</a> &raquo; ' . _t('MEMBERLIST', 'Member List');
 				case 'popularthreads':
 					return '<a href="' . $this->Link() . '">' . $this->Title . '</a> &raquo; ' . _t('MOSTPOPULARTHREADS', 'Most popular threads');
 			}
 		}
 	}
-	
-	
+
+
 	/**
 	 * Get the number of total posts
 	 *
-	 * @return int Returns the number of posts
+	 * @return int Returns the number of posts in this forum holder
 	 */
 	public function getNumPosts() {
-		return Post::get()
-			->innerJoin(ForumHolder::baseForumTable(),"\"Post\".\"ForumID\" = \"ForumPage\".\"ID\"" , "ForumPage")
-			->filter(array(
-				"ForumPage.ParentID" => $this->ID
-			))
-			->count();
+		return $this->getForumStatistics()['Posts'];
```

Comment:
Ha, yes my spidey-sense and a dirty feeling when I wrote this was probably a good indication that this was a non-5.3 feature. Will rework it to be 5.3 happy.